liferay-content-management/liferay-portal
# LPD-26663 Questions are not returned when a space is used in a tag

# What is this trying to solve?

[Link to ticket](https://liferay.atlassian.net/browse/LPD-26663)

This PR fix the bug of the LPD-26663 and add question web module to manage questions tests.

# How am I fixing it?

This PR fix the bug of the LPD-26663 and add question web module to manage questions tests. Now we can use tags with spaces and navigate through them in questions application.

# How can you verify that it works?

The PR includes integration tests.

The PR can also be tested through the following steps :

1. Add a Questions widget to the page
2. Create a new Topic
3. In the new Topic add a new question
4. Add a tag with a space in it for example: “Test Tag“
5. Post the question
6. Go to Tags
7. Click on the tag

Actual Result: No question is returned, as upon clicking on it shows “test-tag“
Expected Result: The question would be returned
